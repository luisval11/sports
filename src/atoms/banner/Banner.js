import React from "react";
import { Card } from "react-bootstrap";

export const Banner = () => {
	return (
		<Card className="m-3 text-center">
			<Card.Body>
				<Card.Title>Football World Cup Score Analysis Board</Card.Title>
			</Card.Body>
		</Card>
	);
};
