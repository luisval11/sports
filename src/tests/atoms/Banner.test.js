import { mount, shallow } from "enzyme";
import { Banner } from "../../atoms/banner/Banner";

describe("Test on Banner", () => {
	test("should match initial snapshot", () => {
		let wrapper = shallow(<Banner />);
		expect(wrapper).toMatchSnapshot();
	});

    test("should has the text 'Football World Cup Score Analysis Board'", () => {
		let wrapper = mount(<Banner />);
        expect(wrapper.find('.h5').text().trim()).toBe('Football World Cup Score Analysis Board');
	});
    
});
