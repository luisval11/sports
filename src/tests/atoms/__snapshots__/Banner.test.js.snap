// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on Banner should match initial snapshot 1`] = `
<Card
  body={false}
  className="m-3 text-center"
>
  <CardBody>
    <CardTitle>
      Football World Cup Score Analysis Board
    </CardTitle>
  </CardBody>
</Card>
`;
